site/src/components/BuildsTable/BuildRow.tsx

```diff
@@ -0,0 +1,139 @@
+import { makeStyles } from "@material-ui/core/styles"
+import TableCell from "@material-ui/core/TableCell"
+import TableRow from "@material-ui/core/TableRow"
+import { WorkspaceBuild } from "api/typesGenerated"
+import { Stack } from "components/Stack/Stack"
+import { useTranslation } from "react-i18next"
+import { MONOSPACE_FONT_FAMILY } from "theme/constants"
+import {
+  displayWorkspaceBuildDuration,
+  getDisplayWorkspaceBuildInitiatedBy,
+} from "util/workspace"
+import { BuildAvatar } from "./BuildAvatar"
+
+export interface BuildRowProps {
+  build: WorkspaceBuild
+}
+
+export const BuildRow: React.FC<BuildRowProps> = ({ build }) => {
+  const styles = useStyles()
+  const { t } = useTranslation("workspacePage")
+  const initiatedBy = getDisplayWorkspaceBuildInitiatedBy(build)
+
+  return (
+    <TableRow
+      hover
+      data-testid={`build-${build.id}`}
+      className={styles.buildRow}
```

Comment:
This is such a nice, tabbable table! I would love to be able to hit 'enter' and navigate to the log page without having to touch my mouse.

Reply:
I was watching TV when I "remembered" that I forgot to add the link 😆 I'm glad you found it as well, really good review 🌮